lib/bots/syncbot.ts

```diff
@@ -106,13 +106,10 @@ export class SyncBot extends ProcBot {
 							service: to.service,
 							flow: to.flow,
 							thread: threadId,
-						}, messenger, data);
+						}, messenger, data)
+						// The upstream expectations are for a promise that resolves to void when complete.
+						.thenReturn();
```

Comment:
Whilst slightly pedantic, we use `.return` in every other CS and TS code, so I'd rather we use it here, too. Ta!